docs/generate.sh

```diff
@@ -21,7 +21,7 @@ OUTPUT_DIR=dcos-commons-gh-pages/dcos-commons
 SWAGGER_CODEGEN_VERSION=2.2.2
 SWAGGER_OUTPUT_DIR=reference/swagger-api
 SWAGGER_JAR=swagger-codegen-cli-${SWAGGER_CODEGEN_VERSION}.jar
-SWAGGER_URL=http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/${SWAGGER_CODEGEN_VERSION}/${SWAGGER_JAR}
+SWAGGER_URL=https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${SWAGGER_CODEGEN_VERSION}/${SWAGGER_JAR}
```

Comment:
Why did we change from `central` to `repo1` here?

Reply:
@kaiwalyajoshi This is because when we reach to [this link](https://central.maven.org/maven2/io/swagger/swagger-codegen-cli/) while using https, it shows us a message to use https://repo1.maven.org. The contents of the pages/directory is same at the replaced host.
Attaching screenshot for the same:
![Screenshot from 2019-06-11 14-36-27](https://user-images.githubusercontent.com/21217038/59259072-b861bd00-8c56-11e9-8a10-d383f538e6a3.png)

Reply:
@kaiwalyajoshi marking this as resolved if it looks fine by your side.